File: app/src/main/java/com/rikkimikki/torrentino/domain/pojo/filmDetailInfo/PromoTrailers.java
package com.rikkimikki.torrentino.domain.pojo.filmDetailInfo;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class PromoTrailers {
    @SerializedName("items")
    @Expose
    private List<PromoTrailersItem> promoTrailersItems;

    public List<PromoTrailersItem> getPromoTrailersItems() {
        return promoTrailersItems;
    }

    public void setPromoTrailersItems(List<PromoTrailersItem> promoTrailersItems) {
        this.promoTrailersItems = promoTrailersItems;
    }
}

File: app/src/main/java/com/rikkimikki/torrentino/domain/pojo/filmDetailInfo/PromoTrailers.kt
package com.rikkimikki.torrentino.domain.pojo.filmDetailInfo

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class PromoTrailers (
    @SerializedName("items")
    @Expose
    val promoTrailersItems: List<PromoTrailersItem?>
)